ios/Classes/API/UsercentricsBannerProxy.swift

```diff
@@ -45,7 +45,7 @@ struct UsercentricsBannerProxy: UsercentricsBannerProxyProtocol {
         } else {
             window = UIApplication.shared.windows.first { $0.isKeyWindow }
         }
-        return window?.rootViewController as? FlutterViewController
+        return window?.rootViewController ?? window?.rootViewController?.presentedViewController
```

Comment:
@mohammadshrahi - Does this new approach cover your use case?

Reply:
I see, I understood the [presentedViewController](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621407-presentedviewcontroller) incorrectly. So what are you proposing? Should we navigate down to get the last `presentedViewController` by itereting until we find a nil?

Reply:
Replacing         `return window?.rootViewController ?? window?.rootViewController?.presentedViewController`
with `window?.rootViewController?.presentedViewController`

Would solve the issue or?